File: DAM_2º-año_1º/Programación/UD11-Ej1/src/Ej2/Ej2.java
package Ej2;

import java.util.HashSet;
import java.util.Random;
import java.util.Set;

public class Ej2 {
    public static void main(String[] args) {
        Random random = new Random();
        Set<Integer> numeros = new HashSet<>();

        while (numeros.size() < 3){
            int numero = random.nextInt(10);
            numeros.add(numero);
        }

        System.out.println(numeros);
    }
}

File: DAM_1/DAM 3er_año_1º-2025-26/Programación/ÚbedaÓscarJunio1/ÚbedaÓscarJunio1/ubedaOscarJunio1/src/main/java/org/example/TerceraEvaluacion/Ej2.kt
package org.example.TerceraEvaluacion

import java.util.stream.IntStream.range

fun esCapicua(numero: Int): Boolean {
    var capicua: Boolean = false;

    val n: String = numero.toString();

    if (n == n.reversed()) {
        capicua = true
    }
    return capicua;
}

fun numerosCapicuaEnRango(inicio: Int, fin: Int): ArrayList<Int> {

    val numeros: ArrayList<Int> = ArrayList();

    for (n: Int in range(inicio, fin)) {
        if (esCapicua(n)){
            numeros.add(n)
        }
    }

    return numeros;
}

fun main() {
    for (n: Int in numerosCapicuaEnRango(1,100)){
        println(n)
    }
}